### PR TITLE
Prepare 3.0.2 release

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -252,34 +300,73 @@ process {
...
+            [   path: { "${params.outdir}/reports/umi/" },
+                mode: params.publish_dir_mode,
+                pattern: "*.{txt}"
+            ]
+        ]
+    }
+
+    withName: 'CALLUMICONSENSUS' {
+        ext.args         = '-M 1 -S Coordinate'
+        ext.prefix       = {"${meta.id}_umi-consensus"}
```
Comment (chris-cheshire) [inline conf/modules.config:361]: mix of underscores and dashes in this filename - you def want this?

**Review by chris-cheshire — APPROVED** [review]
Really impressive work on the testing side, it seems like there is a lot of excellent test coverage on the code base. No other things I can see, are you say - no real structural changes to the pipeline that I can see, just lots of tidying and test improvements.

File: conf/modules.config (hunk 1 above)
Comment (maxulysse) [reply conf/modules.config:361]: I have no issue with mixing `_` and `-` for me `_` would really separate 2 parts of the filename, here the id and the detail of what's in the file; the `-` just joining together 2 parts in that detail.

**Review by FriederikeHanssen — APPROVED** [review]
some of svgs don't have bold writting where they should. Can update this in a separate PR and merge here.

other than that looks great

File: docs/usage.md [hunk 2]
```diff
@@ -15,18 +15,18 @@ Sarek is designed to handle single samples, such as single-normal or single-tumo
 
 The typical command for running the pipeline is as follows:
 
-```console
-nextflow run nf-core/sarek --input samplesheet.csv  --outdir <OUTDIR> -profile docker
+```bash
+nextflow run nf-core/sarek --input samplesheet.csv --outdir <OUTDIR> --genome GATK.GRCh38 -profile docker
```
[the remark this answers is not in the document]
Comment (maxulysse) [reply docs/usage.md:19]: I know, but we do have the `--genome` specified in the TEMPLATE, and I think we're the only pipeline with a default genome...

**Pull request merged**